README.md

```diff
@@ -204,16 +211,24 @@ Section 3.1 with the following exceptions:
   MUST publish on `Ga` a sequence of `group/add-members` messages according to
   [ssb-meta-feeds-group-spec] Section 3.1, such that the union of all recipients
   in that sequence equals all remaining members
-
-It is RECOMMENDED that epoch `G` is the "most preferred epoch" among all the
-epochs that `a` is a member of, which succeed a certain epoch zero.
+>   - TODO what is the `tangles.group` + `tangles.member` for these `group/add-member`
+>   messages. I think it should be
+>     - `tangles.group`:
+>       - root: the epoch zero group/init message id
+>       - previous: latest message
+>     - `tangles.member`:
+>       - root: the group H group/init message id !?
+>       - previous: the previous `group/add-member` message for group H (if it exists)
+    
+It is RECOMMENDED that epoch `H` is the "most preferred epoch" among all the
```

Comment:
:fire: I think this is a typo

Reply:
Ah, I'm sorry, this is not a typo but it's a sentence totally out of context. This was meant for the introduction. I wanted to say that we *recommend* to only perform exclusions on the latest epoch (known locally), not to fiddle with older epochs, although this protocol allows that.